server/src/main/java/org/apache/druid/server/coordinator/duty/CompactSegments.java

```diff
@@ -87,6 +81,9 @@ public class CompactSegments implements CoordinatorCustomDuty
 
   private static final Logger LOG = new Logger(CompactSegments.class);
 
+  private static final Predicate<TaskStatusPlus> TASK_PREDICATE =
```

Comment:
Can we name this `COMPACTION_TASK_PREDICATE`, or `IS_COMPACTION_TASK`?

Reply:
fixed